### DISCUSSION TITLE
shader options

### DISCUSSION THREAD

**Discussion by EMEldd** [IrisShaders/Iris#477] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/86852618/124318490-1c61c180-db4f-11eb-900a-8187fa242e59.png)

hi, this black sky in project luma shaders is a normal thing, easly fixable by touching shader config, but IRIS shader mod for fabric seems to ignore the shader config file, im wondering if its posible to make the "config -> built-in shader zip" or include  some kind of config function on iris shader mod.
thanks

**Comment by WereWolfix** [comment, marked as answer]
I have found a way to change the settings from the shaders files. Obviously you must have unzipped the zip file. I give an example with BSL_shader. Go inside the BSL / shaders / lib folder, inside you will find a file called setting.glsl, open it with a text editor program. All the settings preceded by // will be disabled by default so if you want to have them active you have to delete the // same thing vice versa if you want to disable them add the // in front of the setting

**Reply by EMEldd** [reply]
well luma shaders seems to be more complex than that, but its something thanks, now i can start  testing
